docs/proposals/001-proposal-template.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Neat-enhancement-idea
+authors:
+  - "@sbose78"
```

Comment:
The `@sbose78` in `authors` is a example, right? Should we change it to e.g. `<REPLACE>` to make it more clear?

Reply:
I kept it as "@sbose78" since this is also an enhancement proposal ( of a process ). But I could remove it to avoid confusion. :D

Reply:
Got it. I would add comment since this is a template as well. E.g.

```
authors:
  - "@sbose78" # author github account here
```

Reply:
How about a "co-authors" or "sponsors" tag, so we can capture additional interested parties that might, or might not, have been involved in the initial proposal? A little more skin in the game than just giving a thumbs up.

Reply:
Love the idea of "sponsors"!